tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -5910,12 +5914,12 @@ trait tFoo {
 }
 
 // Comment
-trait tFoo extends /* Comment */ ReflectionMethod {}
+trait tFoo /* Comment */ {}
```

Comment:
traits don't support `extends` and `implements` and no parser throw error on this, so i remove this